install_config/install/advanced_install.adoc

```diff
@@ -1145,8 +1145,22 @@ node1.example.com           Ready                      165d
 node2.example.com           Ready                      165d
 ----
 ====
+
+To verify that the web console is installed correctly, use the master host name and console port number access the console with a web browser:
```

Comment:
To verify that the web console is installed correctly, use the master host name and **_the_** console port number **_to_** access the console with a web browser: